src/union.ts

```diff
@@ -48,60 +55,46 @@ const createFSProxy = (watchers: FSWatcher[]) =>
     },
   );
 
-const fsPromisesMethods = [
-  'access',
-  'copyFile',
-  'open',
-  'opendir',
-  'rename',
-  'truncate',
-  'rmdir',
-  'mkdir',
-  'readdir',
-  'readlink',
-  'symlink',
-  'lstat',
-  'stat',
-  'link',
-  'unlink',
-  'chmod',
-  'lchmod',
-  'lchown',
-  'chown',
-  'utimes',
-  'realpath',
-  'mkdtemp',
-  'writeFile',
-  'appendFile',
-  'readFile',
-] as const;
+export type VolOptions = {
+  readable?: boolean;
+  writeable?: boolean;
+};
+
+class SkipMethodError extends Error {
+  __proto__: Error;
+  constructor() {
+      const trueProto = new.target.prototype;
```

Comment:
This isn't needed, as TypeScript will do it for you.

Reply:
it's still an [issue](https://github.com/Microsoft/TypeScript/issues/13965#issuecomment-278570200) with TS. I was surprised too

Reply:
wow yeah blast from the past I'd completely forgotten about that being a thing when targeting ES5 😬 

Might be worth chucking that link in as a comment above this line just so it's there for future reference, but otherwise all good!